flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/generated/GeneratedInput.java

```diff
@@ -29,6 +29,17 @@
 
 	private static final long serialVersionUID = -7355875544905245676L;
 
+
+	/**
+	 * Creates a GeneratedInput.
+	 *
+	 * @param className class name of the generated Function.
+	 * @param code code of the generated Function.
+	 */
+	public GeneratedInput(String className, String code) {
```

Comment:
I don't think we need it.
You should take `ctx.references`.

Reply:
done